contracts/ExitHandler.sol

```diff
@@ -180,27 +180,40 @@ contract ExitHandler is DepositHandler {
   function finalizeTopExit(uint16 _color) public {
     bytes32 utxoId;
     uint256 exitableAt;
+    Exit memory currentExit;
+
     (utxoId, exitableAt) = getNextExit(_color);
 
-    require(exitableAt <= block.timestamp, "The top exit can not be exited yet");
-    require(tokens[_color].currentSize > 0, "The exit queue for color is empty");
+    for (uint i = 0; i<20; i++) {
+      // if queue is empty or top exit cannot be exited yet, stop
+      if (exitableAt > block.timestamp || tokens[_color].currentSize < 1) {
```

Comment:
> tokens[_color].currentSize < 1

check is unnecessary in the loop. We need to check it once before the loop (to cover "no exits for color" situation, like it was [before](https://github.com/leapdao/leap-contracts/pull/152/files#diff-e8b2bda9737fa1a1ff063bbb22fb4cd5L186)). Later it is covered by condition on [line 211](https://github.com/leapdao/leap-contracts/pull/152/files#diff-e8b2bda9737fa1a1ff063bbb22fb4cd5R211).